docs/authorization.md

```diff
@@ -157,6 +158,29 @@ roleRef:
   name: workspace-admin
 ```
 
+### Initializing Workspaces
+
+By default, workspaces are only accessible to a user if they are in phse `Ready`. Workspaces that are initializing
+can be access only by users that are granted `initialize` verb on the `clusterworkspaces/content` resource in the 
+parent workspace.
```

Comment:
I don't think this is necessarily what we want ...

1. some RBAC (where???) gives you global access to the initializingworkspaces virtual workspace, where you can see all workspaces that need your initializer
2. as a first pass, actions through that workspace impersonate the user who created the workspace

Reply:
as discussed OOB: handling service accounts from parent workspaces won't be handled in this PR.